src/directives/LoadingAnimationDirective.ts

```diff
@@ -47,7 +47,7 @@ export const LoadingAnimationDirective: Directive = {
   unmounted: (el) => {
     const loader = loaders.get(el);
     if (loader) {
-      el.removeChild(loader);
```

Comment:
Is this the same behavior?

Reply:
It is different, the original would throw an error in development during DOM-patching hot reload updates but this didn't seem to happen in other environments/contexts. The notable difference is that even if the loading animation element (`loader`) is not a child of the HTML element corresponding to the Vue component with the loading animation directive, it will be removed. This uncommon scenario seems to only occur in development when the VDOM and DOM are out of sync, and the actual DOM is in an intermediate state when the cleanup function runs.